.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: sphinx-auto-update
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+jobs:
+  script:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get Job URL
+      uses: Tiryoh/gha-jobid-action@v0
+      id: jobs
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        job_name: script
+    - name: checkout with submodule
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+        token: ${{ secrets.PERSONAL_ACCESSTOKEN }}
```

Comment:
Why this needed? It seems the default value of this action is `${{ github.token }}`. I feel no reason to change it to checkout this repository.
https://github.com/actions/checkout